try:
    print('Running the try....')
    print('5'+x)

except Exception as e:

    print(str(e))
    print('Do more things....')

    print('Code conti...')

except NameError as t:
    print(str(t))
    print('NameError Triggered')

    
    
